website/source/docs/http/sys-replication-secondary.html.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: "http"
+page_title: "HTTP API: /sys/replication/secondary"
+sidebar_current: "docs-http-replication-secondary"
+description: |-
+  The '/sys/replication-secondary' endpoint focuses on replication management operations on secondary clusters
+---
+
+# /sys/replication/primary/secondary-token
+
+## GET
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Requires ‘sudo’ capability. Generate a secondary activation token for the cluster with the given opaque identifier.
+  </dd>
+
+  <dt>Method</dt>
+  <dd>GET</dd>
+
+  <dt>URL</dt>
+  <dd>`/sys/replication/primary/secondary-token`</dd>
+
+  <dt>Parameters</dt>
+  <dd>
+    <ul>
+      <li>
+        <span class="param">id</span>
+        <span class="param-flags">required</span>
```

Comment:
We should document if the `ID` needs to be unique. This is important for `revoke-secondary` since otherwise a non-unique ID might mean we are revoking multiple secondaries.